tests/conftest.py

```diff
@@ -13,11 +13,7 @@
 import pytest
 
 from scmdata import ScmDataFrame
-
-try:
-    from pyam import IamDataFrame
-except ImportError:
-    IamDataFrame = None
+from scmdata.pyam_compat import IamDataFrame
```

Comment:
```suggestion
```

Reply:
Using logic to determine if pyam is available from `pyam_compat` module instead of repeating here